src/Sylius/Bundle/AdminBundle/Controller/DashboardController.php

```diff
@@ -41,69 +39,62 @@ final class DashboardController
     /** @var SalesDataProviderInterface|null */
     private $salesDataProvider;
 
+    /** @var StatisticsDataProviderInterface */
+    private $statisticsDataProvider;
+
     public function __construct(
-        DashboardStatisticsProviderInterface $statisticsProvider,
```

Comment:
This is BC break - we cannot merge PR with this. Needs to be reverted

Reply:
According to our [BC Promise](https://docs.sylius.com/en/latest/book/organization/backward-compatibility-promise.html#final-controllers) it is not a BC break if I'm not wrong